examples/experimental/tfe_integration_demo/step2_tfe_private_prediction.ipynb

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
I think we might want to remove this notebook, so we present only one approach (server-client).